lib/router/mbaas-auth.js

```diff
@@ -0,0 +1,29 @@
+const q = require('q');
+
+/**
+ * @param {Object} mediator object used to subscribe and publish read and auth topics
+ * @param {String} username Unique identifier of the user.
```

Comment:
If we are calling it a "Unique identifier of the user", should the variable name not be `userId` ?